packages/engine/apache-arrow-js-bundle/README.md

```diff
@@ -13,7 +13,8 @@ The bundle should appear as `./dist/apache-arrow-bundle.js`
 
 ### Modifying for use in the Engine
 
-- Manually modify the bundle, adding `return arrow` underneath `arrow = __webpack_exports__;` at the very bottom of the file [1]
+- Manually modify the bundle, adding `export` to `var arrow;` at the very top of the file [1]
+- Manually modify the bundle, adding `return arrow` underneath `arrow = __webpack_exports__;` at the very bottom of the file
```

Comment:
Ok I thought it might still be required for the export to work.
Removed in https://github.com/hashintel/hash/pull/540/commits/96afbace7c95470ea2ac21495299fa1eb0077ae4

Reply:
If the tests pass with it removed then that's good :)